integration-test/shadow/shadow_system_test.c

```diff
@@ -659,6 +663,110 @@ static MQTTStatus_t publishToTopic( MQTTContext_t * pContext,
     return mqttStatus;
 }
 
+/**
+ * @brief Subscribes the shadow topics: /update/delta, /update/documents,
+ * /update/accepted, /delete/accepted, /get/accepted, then publish the
+ * regarding payloads to verify if receiving the notification from the
+ * subscribed topics.
+ */
+#define TEST_SEQUENCE() \
```

Comment:
It's not typically in our coding style to create these giant macros. You could define the shadow name as a string and pass it to a test function.

Reply:
Per some offline comments, this won't work, without bigger surgery, because the shadow topic string macros require string literals as arguments.

Reply:
Per discussions, changed to use Shadow_GetTopicString() (instead of the macros) so we could do the subroutine solution.